lib/virt_autotest/utils.pm

```diff
@@ -831,4 +831,30 @@ sub restore_original_guests {
     }
 }
 
+sub upload_virt_logs {
+    my ($log_dir, $compressed_log_name) = @_;
+
+    my $full_compressed_log_name = "/tmp/$compressed_log_name.tar.gz";
+    script_run("tar -czf $full_compressed_log_name $log_dir; rm $log_dir -r", 60);
+    save_screenshot;
+    upload_logs "$full_compressed_log_name";
+    save_screenshot;
+}
```

Comment:
I do not think this function has anything to do virt logs specifically. Maybe you just need a function to upload logs with personally customized name ?